src/prefs/DevicePrefs.cpp

```diff
@@ -203,6 +206,40 @@ void DevicePrefs::PopulateOrExchange(ShuttleGui & S)
       S.EndThreeColumn();
    }
    S.EndStatic();
+
+   if (mProject != nullptr)
+   {
+      S.StartStatic(XO("Quality"));
+      {
+         S.StartThreeColumn();
+
+         auto cmbProjectRate = S.AddCombo(XO("Project rate:"), {}, {});
+
+         for (auto i = 0; i < AudioIOBase::NumStandardRates; ++i)
+            cmbProjectRate->Append(
+               XO("%d").Format(AudioIOBase::StandardRates[i]).Translation());
+
+         cmbProjectRate->SetValue(
+            XO("%d")
+               .Format(static_cast<int>(ProjectRate::Get(*mProject).GetRate()))
+               .Translation());
+
+         cmbProjectRate->Bind(
+            wxEVT_COMBOBOX,
+            [this, cmbProjectRate](auto)
+            {
+               long value;
+               if (cmbProjectRate->GetValue().ToLong(&value))
+                  ProjectRate::Get(*mProject).SetRate(value);
```

Comment:
This event handled changes the project rate but does not change preferences in the .cfg file which is what preference dialog pages are supposed to do.  Furthermore the commit says you “moved” quality but this does not remove it from the other page.

who specified this change?  I think it is not necessary for bears and bars and needs more thought.

Reply:
I think the only other place where we used that project argument in the page factory, is for spectrogram preferences, because this dialog was reused (by me, several years ago) for the new purpose of also changing per-track spectrogram settings.

Reply:
> who specified this change? I think it is not necessary for bears and bars and needs more thought.

@DilsonsPickles did. I agree that this is very questionable change, we have discussed it at some point during the design review.

`ProjectRate` in Audacity is a very confusing thing per se, which is made far worse by this change.

Quality preference page allows you to set the project rate to be used by a new project. This setting is called "Default project rate". 

Selection tool bar, on the other hand, controlled the sampling rate of the project (only by changing the ProjectRate value). This is now moved to Audio Preferences. So there is nothing wrong with what the commit message says. 

I do not understand the value of changing the sample rate of the project without the tracks resampling (or in allowing tracks to have different sample rates in one project). 

I think that we need to discussed it carefully again. I don't think that it is a blocker for 3.3 though. 

@Tantacrul @LWinterberg FYI